src/main/java/edu/cornell/kfs/sys/service/ISOFIPSConversionService.java

```diff
@@ -0,0 +1,15 @@
+package edu.cornell.kfs.sys.service;
+
+public interface ISOFIPSConversionService {
+
+    /*
+     * ISO-to-FIPS mappings are many-ISO-to-one-FIPS.
```

Comment:
In the service interface's comments, the "one" side and "many" side are the opposite of what you stated in the service impl Javadocs. Can you please update or remove these interface comments accordingly?

Reply:
I will remove.